app/views/hyrax/base/_guts4form.html.erb

```diff
@@ -1,5 +1,4 @@
 <% # we will yield to content_for for each tab, e.g. :files_tab %>
-<% tabs ||= %w[metadata files relationships] # default tab order %>
```

Comment:
wondering if we should keep this line to support existing partial overrides on downstream apps, or just cut over now pre-3.0.0.

thoughts?

Reply:
I'm not sure what keeping it would do.  If it has been overridden then I think it doesn't matter because then the app would have a full copy of the partial with that line (or a modified version of it).  I guess in upgrading they would have to reapply their customization through the new helper method.
Were you thinking that downstream apps may have overrides which are calling guts4form without passing tabs and wanting to continue to support them?
Either way, I guess I'm in favor of cutting over now as part of 3.0 to establish the new pattern which engines could rely upon instead of not knowing if local customizations will work with the engine or not.  Does that seem fair?

Reply:
> Were you thinking that downstream apps may have overrides which are calling guts4form without passing tabs and wanting to continue to support them?

yeah exactly.

:+1: to cutting over.